server/cmd/buildbuddy/BUILD

```diff
@@ -31,6 +31,7 @@ container_image(
     base = "@buildbuddy_go_image_base//image",
     symlinks = {
         "config.yaml": "app/server/cmd/buildbuddy/buildbuddy.runfiles/buildbuddy/config/buildbuddy.release.yaml",
+        "buildbuddy": "app/server/cmd/buildbuddy/buildbuddy.runfiles/buildbuddy",
```

Comment:
nit: Can we symlink `buildbuddy` to `/tmp` instead? Asking because runfiles are read only data deps (IIRC) and there might be a chance of clobbering something if we store arbitrary stuff there.

Reply:
(I tested `/tmp` locally and it seems to work)

Reply:
Do we care about it working on restart of the container? `/tmp` destroys its contents on restart on most machines. Could we use `/home/nonroot` instead?

Reply:
nvm, discussed offline, using `/tmp`